sos/policies/distros/redhat.py

```diff
@@ -332,25 +361,37 @@ def upload_sftp(self):
                             " for obtaining SFTP auth token.")
         _token = None
         _user = None
+        if not self.commons['cmdlineopts'].upload_user:
+            _user = self.get_upload_user()
+        else:
+            _user = self.commons['cmdlineopts'].upload_user
```

Comment:
This is redundant; with device auth the upload_user and upload_password is irelevant (until SFTP anonymous user(name) is required to store, dunno from head this detail now).

Reply:
Removed in the next version.